provider/scaleway/scaleway.go

```diff
@@ -65,7 +65,13 @@ func NewScalewayProvider(ctx context.Context, domainFilter endpoint.DomainFilter
 			defaultPageSize = 1000
 		}
 	}
-	p, _ := scw.MustLoadConfig().GetActiveProfile()
+
+	p := &scw.Profile{}
+	c, err := scw.LoadConfig()
+	if err == nil {
+		p, _ = c.GetActiveProfile()
+	}
+
 	scwClient, err := scw.NewClient(
```

Comment:
here you overwrite the error and it's not clear why we don't care about the error.

Reply:
Nothing, but i've reused the same logic from the original PR that has been reviewed but we can change this of course. I can add a log when the file can't be loaded and when the profile was not loaded too if that's okay for you.

Reply:
yes please, I added an example how to make it better. I hope it helps.

Reply:
I keep this thread as it's the one that make most sense from the original idea.
I've added warning in logging because if someone try to use config file and the file contains error he will not be warned about that and can't understand the issue, that's why I think logs are better in the end.
If you don't like the submitted code, feel free to change it as I've don't a lot of knowledge in Go and don't want to waste your time in reviewing and make it hard. Was doing it to help a bit if I can 🙂.

Regards,